### PR TITLE
Chapter 22: XML Drawables

### PR DESCRIPTION
**Pull request by OnicaAdrianCatalin**
## New features

Buttons are customized.

## Changes in code

Added drawable resource for buttons in normal state.

Added drawable resource for buttons in pressed state.

Added drawable resource for integrating the button style in both states.

Screenshot 2022-06-06 at 14 25 38

### CODE REVIEW

File: app/src/main/res/layout/list_item_sound.xml [hunk 1]
```diff
@@ -10,13 +10,19 @@
             type="com.example.beatbox.SoundViewModel" />
     </data>
 
-    <Button
-        style="@style/BeatBoxButton"
+    <FrameLayout
```
Comment (NegreaVlad) [inline app/src/main/res/layout/list_item_sound.xml]: Why do we need the `FrameLayout`?

Comment (OnicaAdrianCatalin) [reply app/src/main/res/layout/list_item_sound.xml]: The book mentioned that if the screen is bigger the recycler view will be stretching our buttons so I wrapped the buttons in FrameLayouts to stretch the FrameLayout, not the button. Now I tested if it really does that stretch and it didn't so I removed the FrameLayout.

Comment (NegreaVlad) [reply app/src/main/res/layout/list_item_sound.xml]: The explanation makes sense to me, why we need the FrameLayout, but if you say it works well without it, we can remove it :)

**Review by NegreaVlad — APPROVED** [review]
LGTM

**Pull request merged**